### DISCUSSION TITLE
Add has_role_# in addition to the current has_role_X

### DISCUSSION THREAD

**Discussion by nep** [ExpressionEngine/ExpressionEngine#4378] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**
It would be nice to be able to refer to roles by their numbers as well as by their short names in the conditional has_role_x.

**Describe the solution you'd like**
For example, imagine that I'm echoing each of these variables in a template.  It would be useful if the final variable also was a "1" -- that way if I rename a role, the template code doesn't need to be changed.

logged_in_primary_role_id: 1
logged_in_primary_role_short_name: super_admins
has_role_super_admins: 1
has_role_1: {has_role_1}

I believe it is accomplishable with a single-line addition to libraries/Template.php at line 4564
```$vars['has_role_' . $role->role_id] = $value;```
or something like that, I don't know, I'm just spit-balling.

**Comment by intoeetive** [comment]
@nep - I think that definitely would not hurt. Please feel free to open a pull request
